### DISCUSSION TITLE
Welcome to Fig Discussions!

### DISCUSSION THREAD

**Discussion by rofr** [DevrexLabs/Fig#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rofr** [comment]
Hey @myty and @Ven0maus thanks for the contributions a few years ago. I've been using Fig in a net framework project ever since and based on feedback (from myself) I decided to simplify things by removing features and cleaning up some code. 

Check out the README and latest release and tell me what you think.

I also shut down the gitter chat and opened up the discussions tab here.

**Reply by myty** [reply]
Awesome. Will check it out!
